app/models/user_group.rb

```diff
@@ -53,6 +53,7 @@ def self.text_search(text)
   def add(users)
     # Force individual queries for paper_trail
     Array.wrap(users).uniq.each do |user|
+      # should we use with_deleted to restore the user_group_member if it was previously deleted?
```

Comment:
Is this preferred?I think we use this pattern for group viewable entities, so I can update this for consistency if desired. Currently it will create a new record if user is removed and re-added to the group.

Reply:
if we have paper trail enabled, then yeah, I vote for `with_deleted` so we re-use the previous member.

Reply:
ok, updated